csrc/ir_internal_nodes.h

```diff
@@ -2271,7 +2276,7 @@ class TORCH_CUDA_CU_API PadOp : public Expr {
  private:
   //! Offset of pad_width inputs in the input vector
   int getPadWidthInputOffset() const {
-    return 1;
+    return 2;
```

Comment:
nitpick: this seems too subtle and a little bit tricky to maintain. But I do realize that it keeps things easier when we update the signature. I'm ok with it.